### DISCUSSION TITLE
nonebot2 是不是不支持python3.11.2？

### DISCUSSION THREAD

**Discussion by follow2015** [nonebot/discussions#324] [问答]
使用nb create 创建项目的时候。出现以下错误
ERROR: Could not find a version that satisfies the requirement nonebot2 (from versions: none)
ERROR: No matching distribution found for nonebot2

**Comment by yanyongyu** [comment]
nb支持，请检查你的pip源

**Comment by follow2015** [comment]
我刚刚重新测试了。直接在Mac下的终端里面创建没啥问题，在vscode下面的终端创建就是上面的错误。vscode里面启用了虚拟环境。

**Reply by yanyongyu** [reply]
有没有完整的截图

**Comment by follow2015** [comment]
![C2060104BD6A50A8D2B304180A2337E3](https://user-images.githubusercontent.com/15902098/220801076-cd2b9c30-e4ce-46e4-8630-8a736398899e.jpg)
就是这个错误。不过现在也没问题了。可能是vscode的原因。把开始的工作目录删除，然后重启了vscode就能正常创建了
